test/arrays/userAPI/assocDistributedHashed.comm-none.good

```diff
@@ -43,7 +43,6 @@ local subdomains:
 {eight, five, four, half, nine, one, pointone, seven, six, ten, three, two, zero}
 
 is empty: false
-find last: (true, half)
```

Comment:
Good point. Grepping around it looks like there are a few other `.good` files I need to update (and it looks like I'll need to make similar updates for the PR that removes `count` as well).

Do you know if I need to run paratests multiple times if I want to ensure that everything works with both `CHPL_COMM=none` and some other comm model (say `CHPL_COMM=gasnet`)?

Or do I just need to build Chapel multiple times with these different configurations and then will `start_tests` automatically cycle through these different configurations (by passing them to the compiler with `--comm`).

I suppose for this specific PR I can just manually run the tests under `userApi` but it'd be good to know for future reference.

Reply:
You may need to take some extra steps in your .bashrc files to make sure things are set up so that paratest hasn't lost some needed environment variables if they aren't something Chapel explicitly tracks, though